service/orders/src/main/java/nl/tudelft/wdm/group1/orders/events/Consumer.java

```diff
@@ -59,7 +61,7 @@ public void consumeOrderProcessedInStockFailed(Order order) {
 
         producer.emitOrderCancelled(order);
 
-        // TODO notify user
+        rest.checkoutFailed(order, new InsufficientStockException("FAILED_DUE_TO_LACK_OF_STOCK"));
```

Comment:
Is it possible to use the OrderStatus Enum for this?

Reply:
Well, it's supposed to be an error message for the user, so it should be some english message. But I couldn't think of a nice message.

Reply:
Ah, fair enough